test/e2e/lazy_activation_policy_test.go

```diff
@@ -90,6 +103,92 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 
+		// Simple Case 3 (Lazy to immediate)
+		// refer: https://github.com/karmada-io/karmada/blob/master/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#simple-case-3-lazy-to-immediate
```

Comment:
Here it's best not to use the master branch, you should use a fixed commit ID instead.

Reply:
```suggestion
		// refer: https://github.com/karmada-io/karmada/blob/fdad87efce1d088c8002856f5ee7586427f1a989/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md?plain=1#L263
```
@chaunceyjiang Is it like this? The new link loses information such as the tag name, leaving only the number of rows. And the original link can find the title according to the tag name, which is relatively more flexible. So I'm more inclined not to modify. What do you think?

Reply:
The code in the master branch may change, which could cause the referenced content to become invalid.

https://github.com/karmada-io/karmada/blob/release-1.9/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#simple-case-3-lazy-to-immediate